src/benchmark/tpch_benchmark.cpp

```diff
@@ -23,6 +21,8 @@
 #include "storage/storage_manager.hpp"
 #include "tpch/tpch_db_generator.hpp"
 #include "tpch/tpch_queries.hpp"
+#include "visualization/lqp_visualizer.hpp"
```

Comment:
Originally, I had this question: "In the example image, where does the unconnected region with `r_name = AMERICA` come from?". Then I found it on the right side of the semi join. So I wondered if that could be made any clearer.

Would it make sense to embed the JoinGraph in the LQP, i.e., replace all involved StoredTableNodes, Predicates, and JoinNodes with the JoinGraph? That way, it would be easier to understand how the two things relate.

Reply:
That would be my idea of a perfect JoinGraphVisualizer, too. But the purpose of this PR is more to port my MT stuff over. Honestly I don't think it's worth the time implementing better visualization for JoinGraphs right now, we got more pressing things to do.